friends/src/App.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import './App.css';
+import axios from 'axios';
+
+import FriendsList from './FriendsList';
+import FriendForm from './FriendForm'
+
+class App extends Component {
+  constructor(props){
+    super(props);
+    this.state = {
+        friends: []
```

Comment:
In this component I see that you are referencing this.state.name, this.state.age, and this.state.email. However, I am not seeing those  in your state. Make they are set to initial values of empty strings in state.